snmp/datadog_checks/snmp/data/default_profiles/_generic-ucd.yaml

```diff
@@ -0,0 +1,136 @@
+metrics:
+  - MIB: UCD-SNMP-MIB
+    symbol:
+      name: cpu.usage
+      OID: 1.3.6.1.4.1.2021.10.1.5.1
```

Comment:
This looks like a columnar metric, will this work and sum it for all entries in the table?

Reply:
Indeed, the OID `1.3.6.1.4.1.2021.10.1.5` correspond to `laLoadInt` column, but `1.3.6.1.4.1.2021.10.1.5.1` correspond to the first row of the column.

I will double check if this makes sense or the whole metric should be converted into a table/symbols definition instead.

Reply:
@ken-schneider Based on the description of the `laLoadInt` and website below. It seems that for OID `laLoadInt`, the elements at index 1, 2, 3, correspond to respectively 1min avg, 5min avg, 15min avg.
So, it seems correct to target the `laLoadInt` at index `1` for our use case.

Sources:

```
laLoadInt OBJECT-TYPE
    SYNTAX	Integer32
    MAX-ACCESS	read-only
    STATUS	current
    DESCRIPTION
	"The 1,5 and 15 minute load averages as an integer.
	 This is computed by taking the floating point
	 loadaverage value and multiplying by 100, then
	 converting the value to an integer."
    ::= { laEntry 5 }
```

And this website: https://www.usenix.org.uk/content/snmp_mib.html

<img width="1010" alt="image" src="https://github.com/DataDog/integrations-core/assets/49917914/8fee1841-9dcf-4be3-bcd7-5fe6ab99efc1">

Reply:
The `laLoad` that function in similar way, has a definition pointing to previous conclusion too:

```
laLoad OBJECT-TYPE
    SYNTAX	DisplayString
    MAX-ACCESS	read-only
    STATUS	current
    DESCRIPTION
	"The 1,5 and 15 minute load averages (one per row)."
    ::= { laEntry 3 }
```